config/initializers/rack_attack.rb

```diff
@@ -154,6 +154,12 @@ def path_without_extentions
     match_data[:account_id] if match_data.present?
   end
 
+  # Prevent abuse of conversation search api
+  throttle('/api/v1/accounts/:account_id/conversations/search', limit: 5, period: 1.minute) do |req|
```

Comment:
The search API which we use at the moment is `search/messages`, `search/conversations` etc